webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -1317,3 +1317,23 @@ export function unacknowledgePost(postId: string): ActionFuncAsync {
         return {data};
     };
 }
+
+export function restorePostVersion(postId: string, restoreVersionId: string, connectionId: string): ActionFuncAsync {
```

Comment:
I know the tests here would be quite trivial, but I wonder if we generally add tests for these.

Reply:
Deferring for now.